dotnet/src/Skills/Skills.Grpc/Extensions/GrpcOperationExtensions.cs

```diff
@@ -21,13 +21,13 @@ internal static class GrpcOperationExtensions
     /// <returns>The list of parameters.</returns>
     public static IReadOnlyList<ParameterView> GetParameters(this GrpcOperation operation)
     {
-        var parameters = new List<ParameterView>();
-
-        // Register the "address" parameter so that it's possible to override it if needed.
-        parameters.Add(new ParameterView(GrpcOperation.AddressArgumentName, "Address for gRPC channel to use.", string.Empty));
-
-        // Register the "payload" parameter to be used as gRPC operation request message.
-        parameters.Add(new ParameterView(GrpcOperation.PayloadArgumentName, "gRPC request message.", string.Empty));
+        var parameters = new List<ParameterView>
+        {
+            // Register the "address" parameter so that it's possible to override it if needed.
+            new ParameterView(GrpcOperation.AddressArgumentName, "Address for gRPC channel to use.", string.Empty),
+            // Register the "payload" parameter to be used as gRPC operation request message.
+            new ParameterView(GrpcOperation.PayloadArgumentName, "gRPC request message.", string.Empty)
+        };
```

Comment:
Not a huge fan of the shorthand here, with the comments iline, and the fact that the param values themselves are method calls... it's a little cluttered. Is this a r# rule? I think, at minimum, a newline between the first value and the second comment would help.

Reply:
Reworked in [that commit](https://github.com/microsoft/semantic-kernel/pull/1914/commits/831905fba119720d71021337e14091de99dd2d05)